src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="iMac"
+      >
+      <h1 class="card__title">
+        Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+      </h1>
+      <div class="card__code">
+        Код товара: 195434
+      </div>
+      <div class="card__feedback feedback">
+        <div class="feedback__stars stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <a href="#" class="feedback__reviews">Отзывов: 5</a>
+      </div>
+      <div class="card__price-block" >
+        <div class="card__price">Цена:</div>
+        <div class="card__value">69 999 грн</div>
+      </div>
```

Comment:
Можно сделать так например))
```suggestion
      <div class="card__price" >
        <div class="card__price-money">Цена:</div>
        <div class="card__price-value">69 999 грн</div>
      </div>
```